src/Template/Vocabulary/add_sentences.ctp

```diff
@@ -55,6 +60,12 @@ $this->set('title_for_layout', $this->Pages->formatTitle($title));
             )
             ?>
         </div>
+        <?php
+            if ($vocabulary->count() == 0) {
+                $noRequestsMessage = __("There are no requests.");
+                echo "<div class='empty-info-text'> $noRequestsMessage </div>";
+            }
+        ?>
```

Comment:
This totally works, but here is an alternative way to put it:
```html
<?php if ($vocabulary->count() == 0): ?>
    <div class="empty-info-text">
        <?= __("There are no requests.") ?>
    </div>
<?php endif; ?>
```

Reply:
Another, more CakePHP-ish way to do it, using the [Html helper](https://book.cakephp.org/3/en/views/helpers/html.html):
```php
if ($vocabulary->count() == 0) {
    echo $this->Html->div('empty-info-text', __('There are no requests.'));
}
```